specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2019-05-05-preview/AlertsManagement.json

```diff
@@ -1032,7 +1039,11 @@
           "description": "Modification details",
           "items": {
             "$ref": "#/definitions/alertModificationItem"
-          }
+          },
+          "x-ms-identifiers": [
+            "oldValue",
```

Comment:
I think modificationEvent is better, and it can be paired with oldValue.
A few events can have the same modification time. But the field and value should be unique enough. (although not 100%)